vm.js

```diff
@@ -3273,318 +3662,23 @@ Object.subclass('Squeak.Primitives',
         }
         return false;
     },
+
+    getPrimitiveFunc: function(primIndex, argCount, primMethod) {
+        if ((primIndex > 264) && (primIndex < 520)) {
+            return this.primitiveFunctions[260];
+        }
+        return this.primitiveFunctions[primIndex];
+    },
+
     doPrimitive: function(index, argCount, primMethod) {
         this.success = true;
```

Comment:
Ah, yes. You need to delete `tryPrimitive()` and replace its callers with calls to `doPrimitive()`, since that now does everything. Which makes much more sense. Great!